web/core/components/gantt-chart/helpers/blockResizables/use-gantt-resizable.ts

```
@@ -0,0 +1,124 @@
+
+    // remove event listeners and call updateBlockDates
+    const handleMouseUp = () => {
+      setIsMoving(undefined);
+
+      document.removeEventListener("mousemove", handleMouseMove);
+      ganttContainerElement.removeEventListener("scroll", handleOnScroll);
+      document.removeEventListener("mouseup", handleMouseUp);
+
+      try {
+        const blockUpdates = getUpdatedPositionAfterDrag(block.id, dragDirection !== "move");
+        updateBlockDates && updateBlockDates(blockUpdates);
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)